qa_forum/models.py

```diff
@@ -0,0 +1,24 @@
+from django.contrib.auth.models import User
```

Comment:
@chuckcwh Is there a reason why you don't break Question and Answer into different files? What if you have 100 models in your project?